### PR TITLE
if currPlay is missing in globalUserId, set cookied value to it

### PR DESCRIPTION
**Pull request by alexunique0519**
### Description

List of proposed changes:

-
-

### CODE REVIEW

File: pages/room/[id].js [hunk 1]
```diff
@@ -107,6 +107,7 @@ export default function Room(props) {
 
   useEffect(() => {
     const currCookie = Cookies.get('userid')
+    console.log('curCookie', currCookie)
```
Comment (krischarbonneau) [inline pages/room/[id].js:110]: Let's remove this before merging

**Review by krischarbonneau — APPROVED** [review]
Works great! Good job :)

**Review by CurtisUnderwood — APPROVED** [review]
Good find!

**Comment by CurtisUnderwood** [comment]
Should note this comes with a warning of memory leak?
image

**Review by MarcoGoC — APPROVED** [review]
Looks good

**Pull request merged**